### PR TITLE
Add sorting by contributions to contributors and rewrite docs

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

**Comment by unho** [comment]
Updated to address raised concerns.

File: pootle/apps/pootle_app/management/commands/contributors.py [hunk 1]
```diff
@@ -44,6 +51,10 @@ def handle_all(self, **options):
         if options["revision"]:
             units = units.filter(revision__gte=options["revision"])
 
+        if options["sort_by"] not in ["name", "contributions"]:
+            # Specified sorting is not valid. Fall back to default.
+            options["sort_by"] = "name"
```
Comment (dwaynebailey) [inline pootle/apps/pootle_app/management/commands/contributors.py]: [``choiced=``](https://docs.python.org/2/library/optparse.html#optparse.Option.choices) in ``make_options`` will do this for free.

**Comment by dwaynebailey** [comment]
Otherwise lgtm

File: pootle/apps/pootle_app/management/commands/contributors.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_app/management/commands/contributors.py]: Investigating that.

**Comment by phlax** [comment]
nice one @unho !

**Pull request merged**